### PR TITLE
Double quotes issue

### LINKED ISSUE #410: Fix content according to the new expressions and string constants syntax [7]

### PR DESCRIPTION
**Pull request by oritstone**
Fixes #410 
- Change the syntax of constant strings to not include double set of quotes
- Change expressions syntax to ${expr}

Signed-off-by: Orit Stone <orit.stone@hp.com>

### CODE REVIEW

**Comment by Bonczidai** [comment]
updated chef content

**Comment by oritstone** [comment]
@Bonczidai please review

**Comment by Bonczidai** [comment]
:+1: looks good

File: content/io/cloudslang/base/os/linux/check_linux_disk_space.sl [hunk 1]
```diff
@@ -82,8 +81,8 @@ flow:
           - return_code
           - returnResult
   outputs:
-    - disk_space: "'' if 'standard_out' not in locals() else standard_out.strip()"
-    - error_message: "'' if 'standard_err' not in locals() else standard_err if return_code == '0' else returnResult"
+    - disk_space: ${ '' if 'standard_out' not in locals() else standard_out.strip() }
+    - error_message: ${ '' if 'standard_err' not in locals() else standard_err if return_code == '0' else returnResult${
```
Comment (Bonczidai) [inline content/io/cloudslang/base/os/linux/check_linux_disk_space.sl]: ${ -> }

File: content/io/cloudslang/base/os/linux/restart_process.sl [hunk 2]
```diff
@@ -49,21 +49,21 @@ flow:
           ssh.ssh_flow:
             - host
             - port
-            - sudo_command: "'echo ' + password + ' | sudo -S ' if bool(sudo_user) else ''"
-            - command: "sudo_command + 'pkill -HUP -e ' + process_name"
+            - sudo_command: "echo ' + password + ' | sudo -S ' if bool(sudo_user) else '"
```
Comment (Bonczidai) [inline content/io/cloudslang/base/os/linux/restart_process.sl]: should be expression

**Comment by Bonczidai** [comment]
builder version is upgraded - let the fun begin

**Comment by Bonczidai** [comment]
http fixed, waiting for consul & docker -- once build passes, then we can start to divide manual testing

**Comment by Bonczidai** [comment]
lane 0 is green

**Pull request merged**